torch/_inductor/codegen/common.py

```diff
@@ -805,8 +819,21 @@ class CSEProxy:
             @staticmethod
             def __getattr__(name):
                 def inner(*args, **kwargs):
+                    # TritonTemplateKernel has no current_node
+                    buf_bounds = ValueRanges.unknown()
+                    if hasattr(V.interpreter, "current_node"):
+                        fx_node = V.interpreter.current_node
+                        buf_bounds = self.bounds.get(fx_node, ValueRanges.unknown())
+                        # Sanity check: The variable is either bounded or unbounded
+                        assert (
+                            buf_bounds is not None
+                            or fx_node in self.bounds.unbounded_vars
```

Comment:
commented this earlier, but it's still a little confusing to me that we could have both `ValueRanges.unknown()` or `None` when both are semantically equivalent. i think it would be clearer to have just one of them

Reply:
removing the assert, as it was a bit stale, given that we were doing a `get` with a `ValueRanges.unknown()` before.